### DISCUSSION TITLE
Changing the Shape of the Scroll Handle

### DISCUSSION THREAD

**Discussion by hasantr** [TomSchimansky/CustomTkinter#1950] [General]
Can we make the shape of the Scroll Handle Rectangular, and the Scroll looks much bigger than its height. Is it possible to change both the Handle size and the Handle shape? Thank you.

**Comment by Akascape** [comment]
@hasantr if you pass `corner radius=0` then the shape of the scroll handle will become rectangular. The width of the handle can be changed passing `width=5`.

**Comment by Aayan-Ali-Hashim** [comment]
from tkinter import Tk, ttk

root = Tk()

style = ttk.Style()
style.configure("Custom.Horizontal.TScrollbar", gripcount=0, troughcolor="red", bordercolor="blue")

scrollbar = ttk.Scrollbar(root, style="Custom.Horizontal.TScrollbar", orient="horizontal")
scrollbar.pack(fill="x")

root.mainloop()
